src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/OffenderController.kt

```diff
@@ -202,4 +202,53 @@ class OffenderController(
   fun getIneligibilityReasons(
     @PathVariable nomsId: String,
   ) = prisonerSearchService.getIneligibilityReasons(nomsId)
+
+  @Tag(name = Tags.OFFENDER)
+  @GetMapping(
+    value = ["/nomisid/{nomsId}/is-91-status"],
+    produces = [MediaType.APPLICATION_JSON_VALUE],
+  )
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @Operation(
+    summary = "Retrieve IS-91 status for offender",
+    description = "Returns IS-91 status for creating a licence for a specific prisoner. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "a boolean for IS-91 status",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = String::class),
```

Comment:
If it is a boolean, should it not return a Boolean class type?

Reply:
Thank you! Corrected 😄